apps/docs/components/LogoDownloadModal.js

```diff
@@ -0,0 +1,193 @@
+import React, { useRef, useEffect } from 'react';
```

Comment:
Please add the header comment here. you can use the eslint auto-fix *or* just copy paste it from another file.